lib/loader/index.d.ts

```diff
@@ -1,118 +1,121 @@
 /// <reference lib="esnext.bigint" />
 
 interface ResultObject {
-  module: WebAssembly.Module,
+  module: WebAssembly.Module
   instance: WebAssembly.Instance
-};
+}
+
+type ImportValue = Function | WebAssembly.Global | WebAssembly.Memory | WebAssembly.Table | number
 
 /** WebAssembly imports with two levels of nesting. */
-interface ImportsObject extends Record<string, any> {
+interface Imports extends Record<string, Record<string, ImportValue>> {
   env?: {
     memory?: WebAssembly.Memory,
     table?: WebAssembly.Table,
+    seed?: Function,
     abort?(msg: number, file: number, line: number, column: number): void,
     trace?(msg: number, numArgs?: number, ...args: number[]): void
-  };
+  }
 }
 
 /** Utility mixed in by the loader. */
 interface ASUtil {
-  memory?: WebAssembly.Memory;
-  table?: WebAssembly.Table;
+  memory?: WebAssembly.Memory
```

Comment:
Semicolons here and below are fine too. Only affected cases are semicolons behind a declaration, like a `function a() {};` or an `interface A {};`.